codetransformer/instructions.py

```diff
@@ -13,6 +42,15 @@ def __get__(self, instance, owner):
         return self._op
 
 
+def _notimplemented_property(name):
+    @property
```

Comment:
Shame you can't do this as `property(abstractmethod(lamba self: raise NotImplementedError(...)))`